VirtualEconomyFramework/VEDriversLite/NFT/NFTHelpers.cs

```diff
@@ -1034,7 +1024,10 @@ public static async Task<string> ChangeNFT(string address, EncryptionKey ekey, I
             try
             {
                 // send tx
-                var rtxid = await NeblioTransactionHelpers.SendNFTTokenAsync(dto, ekey, nutxos);
+                var transaction = await NeblioTransactionHelpers.SendNFTTokenAsync(dto, nutxos);                                
+
+                var rtxid = await NeblioTransactionHelpers.SignAndBroadcastTransaction(transaction, secret);

```

Comment:
Most of the functions has same part from this line + until end of the function. Could you please do some general function for this part?